QRCodeBasedMetroTicketingSystem/QRCodeBasedMetroTicketingSystem.Web/wwwroot/js/home.js

```
@@ -1,41 +1,102 @@
+                submitBtn.removeClass('disabled');
+            },
+            error: function (xhr, status, error) {
+                // Handle error
+                let errorMessage = 'An error occurred while calculating the fare';
+
+                if (xhr.responseJSON && xhr.responseJSON.message) {
+                    errorMessage = xhr.responseJSON.message;
+                }
+
+                // Create error alert
+                let alertDiv = $(`
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'alertDiv' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)